test/json/stake_accounts.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
This file could be in `fixtures/files` so it can be load with `file_fixture`. We could move other files from `json` dir in  separate story.